lib/slack/web/web.ex

```diff
@@ -57,6 +57,9 @@ Enum.each(Slack.Web.get_documentation, fn({module_name, functions}) ->
       end
     end)
 
+    defp get_token(%{token: token}) when is_binary(token), do: token
```

Comment:
Does `when is_binary(token)` violate the principle of least surprise?

eg: we call `get_token(%{token: 123})` it would end up using the default credentials.

I think here it would make more sense to let it blow up than fall-back to the configured token.